packages/react-examples/src/office-ui-fabric-react/TextField/TextField.Basic.Example.tsx

```diff
@@ -22,7 +22,7 @@ export const TextFieldBasicExample: React.FunctionComponent = () => {
         <TextField label="With error message" errorMessage="Error message" />
       </Stack>
       <Stack {...columnProps}>
-        <MaskedTextField label="With input mask" mask="m\ask: (999) 999 - 9999" />
+        <MaskedTextField label="With input mask" mask="m\ask: (999) 999 - 9999" title="A 10 digit number" />
```

Comment:
I think I missed this change in master too--what issue was this fixing? Wondering if `aria-label` (with the label plus instructions) would be more appropriate.

Reply:
I had considered that initially, but upon reading the [HTML input spec](https://html.spec.whatwg.org/multipage/input.html#the-pattern-attribute) opted for the `title` solution, as suggested also by @JustSlone.